app/server/utils/ussd/kenya_ussd_processor.py

```diff
@@ -68,11 +68,12 @@ def replace_vars(menu: UssdMenu, ussd_session: UssdSession, display_text: str, u
             replacements.append(['%exchange_amount%', ussd_session.session_data['exchange_amount']])
         elif 'pin_authorization' in menu.name or 'current_pin' in menu.name:
             if user.pin_failed_attempts() > 0:
-                # TODO: not a great way to do i18n...
+                # TODO(ussd): replace this with i18n placeholders
```

Comment:
this todo being to swap things into the YAML file?

Reply:
swap into the yaml file and use the built in #{} placeholders for variables instead of the final replace_vars business